ec2utils/ec2uploadimg/ec2uploadimg

```diff
@@ -437,41 +442,45 @@ try:
                                               '--type')
 
         key_pair_name = args.sshName
+        ssh_private_key_file = args.privateKey
+        if not key_pair_name and not ssh_private_key_file and not \
+                args.accountName:
```

Comment:
Wrap this condition in (), to match our style, i.e.

if (
    not key_pair_name 
    and not ssh_private_key_file
    and not args.accountName
):